service-worker/prefetch-video/service-worker.js

```diff
@@ -68,7 +70,7 @@ self.addEventListener('install', function(event) {
         // (https://slightlyoff.github.io/ServiceWorker/spec/service_worker/index.html#cross-origin-resources)
         // and it is not possible to determine whether an opaque response represents a success or failure
         // (https://github.com/whatwg/fetch/issues/14).
-        var request = new Request(url, {mode: 'no-cors'});
+        var request = new Request(url/* , {mode: 'no-cors'} */);
```

Comment:
Instead of commenting this out, can you explicitly delete the code? This is a use case in which CORS is required.